client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -139,28 +154,47 @@ const EmailProvidersStackedComparison = ( {
 
 			{ ! isDomainInCart && selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
 
-			<h1 className="email-providers-stacked-comparison__header">
-				{ translate( 'Pick an email solution' ) }
-			</h1>
+			{ header }
 
 			{ selectedSite && (
 				<div className="email-providers-stacked-comparison__sub-header">
-					{ translate( 'Not sure where to start? {{a}}See how they compare{{/a}}.', {
-						components: {
-							a: (
-								<a
-									href={ emailManagementInDepthComparison(
-										selectedSite.slug,
-										selectedDomainName,
-										currentRoute,
-										null,
-										selectedIntervalLength
-									) }
-									onClick={ handleCompareClick }
-								/>
-							),
-						},
-					} ) }
+					{ isDomainInCart
+						? translate(
+								'Not sure where to start? {{a}}See how they compare{{/a}} or {{skip}}skip this step{{/skip}}.',
```

Comment:
It may be a good idea to use more explicit names for components:
```suggestion
								'Not sure where to start? {{compare_link}}See how they compare{{/compare_link}} or {{skip_link}}skip this step{{/skip_link}}.',
```